themes/src/main/resources/theme/keycloak-preview/account/resources/styles.css

```diff
@@ -301,11 +301,35 @@ p.description {
   position: absolute;
   right: 10px;
 }
+.list-view-pf-additional-info .fa-user{
+  margin-right: 5px;
+}
+.linked-account-list .list-view-pf .list-group-item-heading {
+    -ms-flex: 1 0 calc(40% - 20px);
+    flex: 1 0 calc(40% - 20px);
+  }
+.linked-account-list .list-view-pf-additional-info{
+    width:10%
```

Comment:
1, need to add space between the selector and curly braces
2, Include one space after : for each declaration.
3, add ";" end of declaration